### ISSUE TITLE
orange icon in standard dark theme

### ISSUE THREAD

**Issue by marko1231991** [nt1m/livemarks#420]
**Add-on version**
Add-on version: 3.5

**Describe the bug**
I would like to be able to set the orange color of the icon in the firefox dark theme as well

![screenshot_1](https://user-images.githubusercontent.com/109438869/183762410-953f6205-1f82-42c2-a4cd-71b3d212c44c.png)

**Comment by nt1m** [comment]
Oh interesting, this looks like a bug. I'll see if I can fix it. Probably a matter of removing context-fill from the SVG

**Comment by nt1m** [comment]
This actually seems like a Firefox bug, not a bug from this extension. Firefox seems to force a filter on extension icons in dark mode.  https://searchfox.org/mozilla-central/rev/99a9eed20cf195b8ff815604876b6fb73ca5ecd7/browser/themes/shared/urlbar-searchbar.css#360-380

@robwu @emilio Can we remove this? This extension has a setting to configure the extension icon color, and it's not getting honored.

**Comment by nt1m** [comment]
I meant to ping @Rob--W here :)

**Comment by emilio** [comment]
Yeah, +1 on removing this, but it's at least blocked on a container extension release so https://github.com/mozilla/multi-account-containers/pull/2584 and so is picked up...

**Comment by Rob--W** [comment]
Emilio, do we need to block on that update? If an extension has explicitly declared a separate dark theme icon, then I would expect that icon to be used in unmodified form.
